Library/Homebrew/formula.rb

```diff
@@ -922,6 +898,41 @@ def plist_startup
     self.class.plist_startup
   end
 
+  # This method can be used to provide a plist.
+  # @see https://www.unix.com/man-page/all/5/plist/ Apple's plist(5) man page
+  # <pre>plist do
+  #  {
+  #    Label: "#{plist_name}.override",
+  #    ProgramArguments: ["#{opt_bin}/example", "--do-this"],
+  #    RunAtLoad: true,
+  #    KeepAlive: true,
+  #    StandardErrorPath: "/dev/null",
+  #    StandardOutPath: "/dev/null",
+  #  }
+  # end</pre>
+  def self.plist
+    @plists ||= {}
+
+    if block_given?
+      plist = yield
+      plist[:Label] ||= self.plist_name
+      @label = plist[:Label]
+
+      def self.plist_name
+        @label
+      end
+
+      @plists[@label] = plist.to_plist
+    else
+      return @plists
```

Comment:
This `return` is unnecessary in Ruby.

Reply:
(`brew style` will tell you that, `brew style --fix` should fix that).

Reply:
Yeah I am aware Ruby has implicit returns… it just seemed more readable. No problem